client/src/featureform/local_cache.py

```diff
@@ -181,11 +181,19 @@ def get_source_files_for_source(self, source_name, source_variant) -> Set[str]:
             dependencies = json.loads(source["inputs"])
             for name, variant in dependencies:
                 sources.update(self.get_source_files_for_source(name, variant))
+        elif transform_type == SourceType.DIRECTORY.value:
+            path = source["definition"]
```

Comment:
super nit: 

directory = source["definition"]
for file in self.get_files_in_dir(directory):
   sources.add(file)